### PR TITLE
Update README.md

### LINKED ISSUE #72: 3.6 Naming conventions for variables, constants, functions and classes
```js
// for const name we use UPPERCASE
    const CONFIG = {
      key: 'value'
    };
```
I would like to mention that this is legacy since first versions of fortran.
It didn't have const's! 
So for all registries/variables that we didn't wanna ppl to change we started using UPPERCASE notation.
Again: it was just because language didn't support `const`!

It's not `nice` to see that this legacy lives on 30+ years, even when most major languages/compilers understand `const` word. 

Also most editors understand word `const` and display it different than variables (let, var).

I would postulate to drop 30+ years legacy, and rethink how we are doing things, and change:
```js
// for const name we use UPPERCASE
const CONFIG = {
      key: 'value'
};
```
to
```js
const config = {
   key: 'value'
}
```

### PR DESCRIPTION
**Pull request by syzer**
Fixes #72

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -286,13 +286,13 @@
   // for class name we use UpperCamelCase
   class SomeClassExample { 
     
-    // for const name we use UPPERCASE
-    const CONFIG = {
-      key: 'value'
+    // for const name we use const keyword
```
Comment (refack) [inline README.md]: names

Comment (syzer) [reply README.md]: ?

**Comment by refack** [comment]
Hello @syzer and thank you for the contribution 🥇 
Maybe we should consider merging those two paragraphs (all are variables, and named in the same style)?

File: README.md (hunk 1 above)
Comment (refack) [reply README.md]: `for const names we use the const keyword`
(either `a const name` or `const names`)

Comment (syzer) [reply README.md]: https://github.com/i0natan/nodebestpractices/pull/82

**Comment by i0natan** [comment]
@refack I'm not sure whether we fixed the permission issue - can you know merge PR?

@syzer - thanks for this nice contribution :)

**Comment by syzer** [comment]
Closing in favour of #82

**Pull request closed without merging**